guides/wordpress-openid-connect-client-plugin-integration/index.md

```diff
@@ -0,0 +1,107 @@
+
+You can also visit the plugin’s page to download, install and activate manually if you prefer:
+
+[wordpress.org/plugins/miniorange-openid-connect-client/](https://wordpress.org/plugins/miniorange-openid-connect-client/)
+
+## Step 3. Configure the plugin
+
+Once activated, the plugin creates a menu option called ‘miniOrange OpenID Connect’.  Click on this to start configuring the plugin to work with DID.
+
+![screenshot of miniOrange OpenId connect plugin menu option](/assets/images/wordpress-guide/plugin-in-menu.jpg)
+
+You will be presented with a screen which looks like this:
+
+![screenshot of miniOrange OpenId connect plugin config dashboard](/assets/images/wordpress-guide/miniOrange-config-panel.jpg)
+
+Scroll right to the bottom on this page and look for the option called ‘__Custom OpenID Connect App__’ under the heading ‘Custom Applications’.  It looks like this:
+
+![screenshot of miniOrange OpenId connect plugin config dashboard](/assets/images/wordpress-guide/custom-applications.PNG)
+
+Select ‘Custom OpenID Connect App’.  You will now see this screen:
+
+![screenshot of miniOrange OpenId connect plugin config dashboard](/assets/images/wordpress-guide/plugin-config.jpg)
+
+Complete the fields using the following values:
+
+```
+App Name: DID
+Client ID: *copy and paste this from your DID app settings page.*
+Client Secret: *copy and paste this from DID app settings page.*
+Scope: openid
+Authorize Endpoint: https://auth.did.app/oidc/authorize
+Access Token Endpoint: https://auth.did.app/oidc/token
+
+```
+Make sure these options are set as follows:
+
+1. Set client credentials in header: checked.
+2. Set client credentials in body: unchecked.
+3. Show on login page: checked.
+
+Finally, press ‘__Save Settings__’.
+
+DID’s integration with this plugin is now complete.  A ‘Sign in with DID’ button now appears on the Wordpress sign in page:
+
+![screenshot of logging into Wordpress with DID](/assets/images/wordpress-guide/login-with-did.PNG)
+
+## Optional: Adding Login Buttons to your Wordpress Template
+
+You can add ‘Login with DID’ buttons elsewhere on your Wordpress website using the plugin.  To do this, choose the menu options: __Appearance > Widgets__.
+
+In the example Wordpress theme we are using, we have been able to add a login button to ‘Footer 1’.  You can add the Login button widget to any dynamic part of your website templates by selecting the __miniOrange Open ID Connect__ widget and clicking ‘add widget’ as per the screenshot below.  The exact appearance of this will vary depending on your Wordpress theme.
+
+![screenshot of adding login buttons to Wordpress template content blocks](/assets/images/wordpress-guide/widget1.PNG)
+
+## Optional: Login Button Styling
+
+The ‘Login with DID’ button can be styled with some simple CSS rules.  The following CSS can be used to style the button with DID’s colour scheme:
+
+```
+.mo_oauth_login_button {
+display: block;
```

Comment:
Can you fix the indentation on this code block.

Reply:
yes absolutely